### ISSUE TITLE
Incorrect predictions from pretrained voc weights yolo

### ISSUE THREAD

**Issue by Ajaz-Ahmad** [simo23/tinyYOLOv2#15]
Hi simo23,

I was inspired with your code and have included yolo network with batch normalization. Though the model loads successfully but it gives poor predictions. Here is the link to the code, Can you help me?

https://github.com/Ajaz-Ahmad/YOLO-WeightLoader

Regards
